docs/operations/mediated_devices_configuration.md

```diff
@@ -0,0 +1,130 @@
+# Automatic configuration of mediated devices / vGPUs
+
+KubeVirt aims to facilitate the configuration of mediated devices on large clusters.
+The new `mediatedDevicesConfiguration` API in KubeVirt CR allows administrators to
+create or remove mediated devices in a declarative way, by providing a list of the desired mediated devices types  that they expect to be configured in the cluster.
+
+It would also be possible to provide a more specific configuration that targets a specific node or a group of nodes directly.
+This can be achieved by using the `nodeMediatedDeviceTypes`, however, it must be used in a combination with `mediatedDevicesTypes`, to override the global
+configuration set in `mediatedDevicesTypes` section. Therefore, using `nodeMediatedDeviceTypes` without the `mediatedDevicesTypes` section is not supported.
+
+
+KubeVirt will use the provided configuration to automatically create the relevant mdev/vGPU devices on nodes that can support it.
+
+Currently, a single type per card will be configured.
+The maximum amount of instances of the selected mdev type will be configured per card.
+
+> Some vendors, such as NVIDIA, require a driver to be installed on the nodes to provide mediated devices, including vGPUs.
```

Comment:
Nit: This line is just displayed with a weird indent, I was hoping for something more obvious like a "note" box or something...

Reply:
+1 Done